### ISSUE TITLE
False positive for `import/no-duplicates` when dealing with .d.ts extended only import like `vue-router/auto-routes`

### ISSUE THREAD

**Issue by nekomeowww** [un-ts/eslint-plugin-import-x#372]
## Summary

Failed to check source code:

```
import { createRouter, createWebHistory } from 'vue-router'
import { routes } from 'vue-router/auto-routes'
```

Reports:

```
  2:52  error  '~/path/to/project/node_modules/vue-router/dist/vue-router.mjs' imported multiple times  import/no-duplicates
  3:24  error  '~/path/to/project/node_modules/vue-router/dist/vue-router.mjs' imported multiple times  import/no-duplicates
```

The `/auto-routes` wasn't exported by `vue-router`, but extended from `unplugin-vue-router/client` from [`unplugin-vue-router`](https://github.com/posva/unplugin-vue-router), a Vite/Webpack plugin for generating compile-time only, the types it extended was coming from here: https://github.com/posva/unplugin-vue-router/blob/f2fb721b2ab6086160e545cac02c87e181ee0db8/client.d.ts#L1C1-L30C2 .

## Reproduction

https://stackblitz.com/edit/vitejs-vite-mgbv38xp?file=eslint.config.mjs

**Comment by JounQin** [comment]
Please search before rasing new issues.

And this is not a false postive in our view:

https://app.unpkg.com/vue-router@4.5.1/files/package.json#L10-26

https://app.unpkg.com/vue-router@4.5.1/files/package.json#L30-46

Disable comments are just fine enough for such cases.
